### PR TITLE
n7k atherton hsrp

### PR DESCRIPTION
**Pull request by saichint**
This PR is for adding hsrp support for n7k running atherton image. The README.md file is NOT changed yet, as there are no references to atherton in that. But it needs to be specified that this hsrp support is only for n7k running atherton. Please provide any comment on how to show that in the caveats.

### CODE REVIEW

**Comment by mikewiebe** [comment]
@saichint Given that we don't know exactly what the Atherton version number will be, just add the following in the caveats table and we will update it with the exact version string later.

`Supported in OS Version 8.0 and later on N7k`

File: examples/cisco/demo_hsrp.pp [hunk 1]
```diff
@@ -47,6 +56,12 @@
       hsrp_version       => 2,
     }
 
+    cisco_command_config { 'ipv6-addr':
```
Comment (mikewiebe) [inline examples/cisco/demo_hsrp.pp]: Why do we need to use cisco_command_config here to add the ipv6 address?  Can't we use cisco_interface for this?

Comment (saichint) [inline examples/cisco/demo_hsrp.pp]: I could not find any property with which I can set ipv6 address in the cisco_interface provider. I do see ipv4 address but no ipv6. If I am missing something, please correct me and point me to the right property.

Comment (mikewiebe) [inline examples/cisco/demo_hsrp.pp]: @saichint your right.  We don't have support for ipv6.  Never mind.

**Comment by mikewiebe** [comment]
👍

**Pull request merged**